README.md

```diff
@@ -81,7 +81,10 @@ Build the Surefire project using **Maven 3.2.5+** and **JDK 1.8+**.
      * Check module `surefire-grouper` has source folder `target/generated-sources/javacc`.
        If not, add it manually in the module's project properties
 
-* Setup for development in [IntelliJ IDEA](https://www.jetbrains.com/idea/) should work out of the box.
+* Setup for development in [IntelliJ IDEA](https://www.jetbrains.com/idea/) works largely out of the box. There may be unresolved symbols because [IntelliJ does not support the maven-shade-plugin](https://youtrack.jetbrains.com/issue/IDEA-126596). As a workaround:
+
+    * In IntelliJ, right click `surefire-shared-utils/pom.xml` and choose "Maven" -> "Ignore Projects". Don't remove the module if prompted.
+    * Right click the parent `pom.xml` and choose "Maven" -> "Reload project"
```

Comment:
Steps for Eclipse are also needed for InteliJ 
 - install ... shared
 - compile ... grouper

After it you need select `ide-development` in Maven project and refresh

![image](https://user-images.githubusercontent.com/3578921/198565030-e1998ada-8eeb-44a7-b5cd-d0b26e8504f4.png)

Reply:
Ignoring module is not needed.

Reply:
Hey, thanks for the feedback! I checked, and indeed your instructions also work. W.r.t. this:
> * Setup for development in [IntelliJ IDEA](https://www.jetbrains.com/idea/) should work out of the box.

I propose changing it to reflect these setup steps. I'll update my PR.

Reply:
@slawekjaranowski I updated the PR to reflect these instructions, please have a look :)

Reply:
Thanks.
Please also check instruction by second hands 😄 
You can before test clean .m2/repository to be sure that works from scratch.